Document/0x05f-Testing-Local-Authentication.md

```diff
@@ -250,6 +252,10 @@ Android Oreo (API 26) adds two additional error-codes:
 - `FINGERPRINT_ERROR_LOCKOUT_PERMANENT`: The user has tried too many times to unlock their device using the fingerprint reader.
 - `FINGERPRINT_ERROR_VENDOR` – A vendor-specific fingerprint reader error occurred.
 
+##### Third party SDKs
+
+Make sure that fingerprint authentication and/or other types of biometric authentication happens based on the Android SDK its APIs. If this is not the case, ensure that the SDK have been propperly vetted for any weaknesses, next: make sure that there is no "pass/no pass" setup. In all cases: the biometric APIs should bind to other means (e.g. unlock a secret or a cryptographic operator) which cannot be touched when the biometric authentication itself is bypassed.

```

Comment:
Put an "and" after SDK

Reply:
that the SDK has (instead of have)

Reply:
What do you mean by that "next: make sure that there is no "pass/no pass" setup."?
Do you mean that the pass/no pass shouldn't be done within the App but instead in a secure hardware like the Secure Enclave should decide if the biometric auth was passed (as the attack described in objection, to bypass touchID, https://github.com/sensepost/objection/wiki/Understanding-the-TouchID-Bypass)